jena-fuseki2/jena-fuseki-ui/src/views/dataset/Edit.vue

```diff
@@ -237,13 +238,19 @@ export default {
       this.content = newVal
       this.codemirrorEditor.scrollTo(scrollInfo.left, scrollInfo.top)
     },
-    listCurrentGraphs: async function () {
+    listCurrentGraphs: async function (count=true) {
       this.loadingGraphs = true
       this.loadingGraph = true
       this.code = ''
       this.selectedGraph = ''
       try {
-        this.graphs = await this.$fusekiService.countGraphsTriples(this.datasetName, this.services.query['srv.endpoints'][0])
+        const endpoint = this.services.query['srv.endpoints'][0]
+        if (count) {
+          this.graphs = await this.$fusekiService.countGraphsTriples(this.datasetName, endpoint)
+        } else {
+          const graphs = await this.$fusekiService.listGraphs(this.datasetName, endpoint)
+          this.graphs = Object.fromEntries(graphs.map(name => [name, ""]))
+        }
```

Comment:
I will try to split this function into two later :+1:

Reply:
Yeah, it looks like one function with a bool=true returns just the count, and for bool=false the graphs (with the counts too?). I think one is bringing the count to display the graphs and the count. The other is bringing the graphs and services available. If so, we can probably split this function into two, one for the graph services, and another for the counts. (Need to confirm)